src/Components/EditData.js

```diff
@@ -0,0 +1,386 @@
+import React from "react";
```

Comment:
Most of the EditData component is duplicated, so try to reuse the AddStudent component for edit as well